docs/admin/troubleshooting/jcmd/docker.rst

```diff
@@ -269,5 +243,21 @@ Java Flight Recorder (JFR)
    is not "blown up".
 
 
+Notes
+=====
+
+On earlier versions of the CrateDB OCI image, you needed to use ``chroot`` to
```

Comment:
Isn't this up to the docker version instead of the image? `--user` is a docker command option

Reply:
Thanks! It also works when using `su` instead of `chroot`. I will amend the patch correspondingly.

Reply:
I've tried to improve the wording with f60c48328.